indica/examples/example_diagnostic_models.py

```diff
@@ -28,11 +28,6 @@ def run_example_diagnostic_model(
     transform = transforms[instrument]
     transform.set_equilibrium(equilibrium)
 
-    kwargs = {}
```

Comment:
if you don't have this, the pinhole camera model doesn't work - if you delete it from here, it should go in the example_ methods that call that model